### PR TITLE
Fixes for beta 4

### PR DESCRIPTION
**Pull request by emanuele45**
A couple of easy fixes.

### CODE REVIEW

**Comment by emanuele45** [comment]
@Spuds since you are more familiar with selenium, does this error log give you any hint on what is going bad?
https://travis-ci.org/emanuele45/Dialogo/jobs/172689785
It's the same build as the one of this PR:
https://travis-ci.org/elkarte/Elkarte/builds/172690096
but done on my repo when I push.
I'm not sure why it passes here and not there. But if I restart the job it gets green-light.

File: themes/default/css/_besocial/index_besocial.css [hunk 1]
```diff
@@ -2066,6 +2066,9 @@ a.help .icon {
 		box-shadow: 3px 3px 3px rgba(0, 0, 0, 0.3);
 	}
 
+	#main_menu .linklevel1.active, #main_menu .linklevel1.active:hover {
+		border: 0px;
```
Comment (Vekseid) [inline themes/default/css/_besocial/index_besocial.css]: border: 0; or border: none; would be preferable to try to cut down on px instances.

Comment (emanuele45) [inline themes/default/css/_besocial/index_besocial.css]: I changed that to transparent according to Spuds suggestion: http://www.elkarte.net/community/index.php?topic=3689.msg29449#msg29449

**Comment by emanuele45** [comment]
I increased the SensioLabsInsight sensibility in order to avoid these red crosses.
At the moment it's just an informational tool, not really something we can use to judge the situation (we already know there is **a lot** to improve as usual :P)

File: sources/subs/Recent.subs.php [hunk 2]
```diff
@@ -268,4 +268,121 @@ function earliest_msg()
...
+ * Callback-function for the cache for getLastTopics().
+ *
+ * @param mixed[] $latestTopicOptions
+ */
+function cache_getLastTopics($latestTopicOptions)
+{
+	return array(
+		'data' => getLastTopics($latestTopicOptions),
+		'expires' => time() + 60,
+		'post_retri_eval' => '
```
Comment (live627) [inline sources/subs/Recent.subs.php]: You still use `post_retri_eval`? Might be best to use closures for this.

File: sources/subs/Recent.subs.php [hunk 3]
```diff
@@ -268,4 +268,121 @@ function earliest_msg()
...
+	);
+}
+
+/**
+ * Get the latest posts of a forum.
+ *
+ * @param mixed[] $latestTopicOptions
+ * @return array
+ */
+function getLastTopics($latestTopicOptions)
```
Comment (live627) [inline sources/subs/Recent.subs.php]: Should merge this with ssi_recentTopics() where the query us already split.

**Comment by live627** [comment]
> A couple of easy fixes.

*yawn*

Continually checking on this thread finally paid off.

File: sources/subs/Recent.subs.php (hunk 2 above)
Comment (emanuele45) [inline sources/subs/Recent.subs.php]: > Notice: Serialization of 'Closure' is not allowed

The post_retri_eval is stored in the cache, that's the only reason I still have to ditch it.

**Comment by emanuele45** [comment]
Well... I merge this one. :stuck_out_tongue:

**Pull request merged**